legacy/firmware/config.c

```diff
@@ -606,58 +609,17 @@ bool config_getU2FRoot(HDNode *node) {
   return ret;
 }
 
-bool config_getRootNode(HDNode *node, const char *curve, bool usePassphrase) {
-  // if storage has node, decrypt and use it
-  StorageHDNode storageHDNode = {0};
```

Comment:
is this going away because XPRV support is gone?

Reply:
Indeed. At least that is what I am thinking. And also @prusnak mentioned it himself

Reply:
Correct. AFAIK we don't support loading XPRV on TT, so let's drop this functionality from T1 as well.

Reply:
it's actually gone as of fd8aed610543d38f8f01091c2badd8af32c1d5f2, this code seems to be vestigial

Reply:
Indeed. Marking as resolved.